src/ape/managers/query.py

```diff
@@ -1,13 +1,42 @@
-from typing import Dict, Optional
+from eth_utils import keccak, to_hex, to_bytes
+from functools import partial
+from typing import Any, Dict, Optional
 
 import pandas as pd
+from pydantic import BaseModel
 
 from ape.api import QueryAPI, QueryType
+from ape.api.query import ContractEventQuery, BlockQuery, _BaseQuery
 from ape.exceptions import QueryEngineError
 from ape.plugins import clean_plugin_name
 from ape.utils import ManagerAccessMixin, cached_property
 
 
+def get_columns_from_item(query: _BaseQuery, item: BaseModel) -> Dict[str, Any]:
+    return {k: v for k, v in item.dict().items() if k in query.columns}
+
+
+class DefaultQueryProvider(QueryAPI):
```

Comment:
If this class needs docs, make sure to add those (and the methods too).

Reply:
Was waiting to get a pass before adding docs

Reply:
Oh wait, I forgot that if you have docstrings on the base class, then the docs get propagated upwards.

Remove the docs on the methods, and just add one on the class at the upper level describing the unique properties of the default provider

Reply:
@unparalleled-js why couldn't we remove the docs from the methods again?